packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -78,6 +78,10 @@ const BlockActionsMenu = ( {
 		clipboardBlock &&
 		canInsertBlockType( clipboardBlock.name, rootClientId );
 
+	const [ blocks ] = useEntityBlockEditor( 'postType', 'wp_block', {
+		id: reusableBlock.attributes.ref,
+	} );
```

Comment:
I would recommend using a similar method as in https://github.com/WordPress/gutenberg/pull/45819/files#r1024781810.

Reply:
@Mamaduka i've made the suggested changes, but I think I will nuke this PR, and redo it from scratch.

Reply:
Hi @Mamaduka, thanks very much for your recommendations! I've made them. I also recreated the branch since the original got messed up, so the very first changes may not be accessible.